src/components/projects/new/project-image-picker.svelte

```diff
@@ -9,8 +10,11 @@
   let promise = null;
   let dataURL = null;
   let error = null;
-  let fileReader = new FileReader();
-  fileReader.onloadend = () => dataURL = fileReader.result;
+  let fileReader;
+  onMount(() => {
+    fileReader = new FileReader();
+    fileReader.onloadend = () => dataURL = fileReader.result;
+  });
```

Comment:
Why did it need to be wrapped in `onMount`? It was working without it

Reply:
It wasn't. It was working for Create Project because this was never rendered on the server there, now it is.